src/ui/mainformview.js

```diff
@@ -56,17 +71,16 @@ export default class MainFormView extends View {
 			this.mathView = new MathView( engine, lazyLoad, locale, previewUid, previewClassName );
 			this.mathView.bind( 'display' ).to( this.displayButtonView, 'isOn' );
 
-			children = [
-				this.mathInputView,
-				this.displayButtonView,
-				this.previewLabel,
-				this.mathView
-			];
-		} else {
-			children = [
-				this.mathInputView,
-				this.displayButtonView
-			];
+			children.push( this.previewLabel, this.mathView );
+		}
+
+		if ( this.mathLiveEnabled ) {
+			this.mathLiveView = new MathLiveView( locale, mathLiveSettings );
+			this.mathLiveView.on( 'input', event => {
+				this.mathInputView.inputView.value = event.source.value;
+				this.mathInputView.inputView.fire( event );
```

Comment:
It looks a bit risky to use internals of `mathInputView`. Can we instead use the `equation` setter which does that for us?
```suggestion
				this.equation = event.source.value;
```

Reply:
Docs kinda say it in https://ckeditor.com/docs/ckeditor5/latest/framework/guides/architecture/ui-library.html#best-practices